### ISSUE TITLE
shipit_static_analysis: Repeated review for already-dropped issues

### ISSUE THREAD

**Issue by squelart** [mozilla-releng/services#663]
**MozReview URL:** https://reviewboard.mozilla.org/r/174110/#review192572

**Problem:** The exact same issue was already reported just above ( https://reviewboard.mozilla.org/r/174110/#review192124 ), but I decided to "drop" it then, so it would be great if SSA could recognize that and not comment again.

**Comment by jankeromnes** [comment]
Indeed, our bot currently doesn't take into account previous reviews, and thus is will report any dropped issues again after each push. This is annoying and counter-productive.

If we accept that our bot may always generate false positives, we should get better at handling them, e.g. by detecting when a particular defect was dropped by a developer, and maybe not report it again in future pushes. (Note that this probably involves identifying defects across changes, which is a non-trivial problem, so we'll probably have to develop heuristics and iterate on them to improve their results).

**Comment by sylvestre** [comment]
example with phabricator: https://phabricator.services.mozilla.com/D4899#inline-18963
